### ISSUE TITLE
Column Size Issue when only fixed width columns

### ISSUE THREAD

**Issue by alexander-alvarez** [offirgolan/ember-light-table#260]
There seems to be an issue when you only specify fixed sized columns, where the widths you specify aren't being respected because there isn't another column to 'absorb' the rest.

See:
https://github.com/offirgolan/ember-light-table/compare/master...alexander-alvarez:with-column-issue

![image](https://cloud.githubusercontent.com/assets/11779993/20194329/d76d8b2c-a75f-11e6-9e06-e603f7f1d973.png)

not sure if this has anything to do with:

`If COL or COLGROUP elements are present, they specify the number of columns and the table may be rendered using a fixed layout. Otherwise the autolayout algorithm described below should be used.`

see section `B.5.2 Recommended Layout Algorithms` from:
https://www.w3.org/TR/html401/appendix/notes.html#notes-tables

**Comment by alexander-alvarez** [comment]
bump @offirgolan 
Any insights to share -- seems like something that was intentionally avoided in the demo?

**Comment by offirgolan** [comment]
Hi @alexander-alvarez, sorry for the delay. I'm currently traveling out of the country so my response times are / will be quite slow for the next few weeks. 

I noticed this issue a couple weeks ago and was able to fix it in #222 but I'm not sure if that's the best solution. 

> see section B.5.2 Recommended Layout Algorithms from:
https://www.w3.org/TR/html401/appendix/notes.html#notes-tables

This is HTML4 documentation which I'm not sure is too valid any more. Most of the COL and COLGROUP properties have been deprecated as well so maybe I'm missing something here. 

@taras @thoov do you guys have any ideas on this?

**Comment by alexander-alvarez** [comment]
No problem -- I'll take a look at other table libraries and see what they do to support the equivalent of `totalWidth` property and report back.

**Comment by alexander-alvarez** [comment]
seems some other implementations do the same calculation: https://github.com/facebook/fixed-data-table/blob/master/src/FixedDataTableRow.react.js#L151 but only allow for pixel widths.

Want me to take a look at that right scroll PR in the downtime and see how to get ember-scrollable to show? I know there were issues with scrollable only supporting L-R or Top-Bottom, not both, but maybe as a step 1 we get it to work with 2 containers, and find a way to bridge the two?

**Comment by alexander-alvarez** [comment]
maybe this snippet could help us?
https://github.com/alphasights/ember-scrollable/blob/master/addon/components/ember-scrollable.js#L76

**Comment by offirgolan** [comment]
@alexander-alvarez it seems like horizontal support is getting more and more attention. As I'm currently on vacation I have very limited internet access so if you have the time to see if you can get ember-scrollable working in both directions that would be great. If you are unsuccessful, I can just merge #222 to unblock folks and hopefully find a better solution further down the road.

**Comment by alexander-alvarez** [comment]
I starting looking into the ember-scrollable repo two days ago, will continue working over there during this holiday and see what comes out of it

**Comment by offirgolan** [comment]
Thank you @alexander-alvarez 😄

**Comment by offirgolan** [comment]
Resolved in #222. (I'll make a new issue to track the ember-scrollable progress)

**Comment by trumb1mj** [comment]
This has actually popped up again.  I think it's related to a new version of ember-scrollable but not sure..?  Can this be reopened?
